strava-connector/README.md

```diff
@@ -0,0 +1,67 @@
+# Strava Activities Connector
+
+*This is not an official Google product*
+
+This [Data Studio] [Community Connector] queries Strava activity data for an
+authenticated user.
+
+This connector uses the [Strava API].
+
+## Deploy the Community Connector yourself
+
+To deploy this community connector:
+
+1. Add the OAUTH2 library to your script.
```

Comment:
If devs are using the manifest from this repo, they don't have to separately add the Oauth2 library since libraries are included via the manifest.